src/biokbase/narrative/jobs/batch.py

```diff
@@ -274,7 +274,9 @@ def generate_input_batch(app, tag='release', **kwargs):
     batch_inputs = list()
     param_ids = input_vals.keys()
     product_inputs = [input_vals[k] for k in param_ids]
-    batch_size = reduce(lambda x, y: x*y, [len(p) for p in product_inputs])
```

Comment:
Appreciate you refactoring out the map and reduce expressions

Reply:
👍  (they didn't work anyway 😬)